packages/d3fc-webgl/src/glDraw.js

```diff
@@ -0,0 +1,91 @@
+import attributeBuilder from './buffers/attributeBuilder';
+import glScaleBase from './scale/glScaleBase';
+import programBuilder from './program/programBuilder';
+
+export default () => {
+    let data = null;
+    let context = null;
+    let changed = false;
+    let program = programBuilder();
+    let xScale = glScaleBase();
+    let yScale = glScaleBase();
+    let decorate = () => {};
+
+    const firstDraw = () => {
+        program.buffers().attribute('aVertexPosition', attributeBuilder(data));
```

Comment:
`aVertexPosition` is used twice, we should put it into a const, perhaps in a different file so that we could add any other shader name definitions in the same place

Reply:
I'd been toying with the idea of having some kind of metadata for shader snippets - essentially at the moment to use a shader snippet you need to know exactly which attributes/uniform names are required.
This may be a good solution for that.

Reply:
In the meantime I'll just pull the name to a const though.